frontend/src/models/Cluster.ts

```diff
@@ -0,0 +1,19 @@
+import { Criteria } from './types/criteria.ts';
```

Comment:
I think in general, we don't need a cluster data structure.

I'm thinking it would make more sense to instead make a module for local user data (and it could be expanded to our users feature in the future)

it would essentially store "favorite" criteria clusters as a field along with any other user-specific things we want to implement (maybe  a dark/light mode, other preference type things)

Reply:
@cmccand1 @msanc156  what do you guys think about that?